src/main/java/org/opendaylight/infrautils/simple/ReadyWiring.java

```diff
@@ -9,13 +9,13 @@
 
 import com.google.inject.AbstractModule;
 import org.opendaylight.infrautils.inject.PostFullSystemInjectionListener;
-import org.opendaylight.infrautils.ready.SystemReadyBaseImpl;
 import org.opendaylight.infrautils.ready.SystemReadyMonitor;
+import org.opendaylight.infrautils.ready.spi.SystemReadyImpl;
```

Comment:
ATTN https://git.opendaylight.org/gerrit/#/c/76532/ change it to `SimpleSystemReady`